mysql_schema_configuration/mysql_schema_configuration.tf

```diff
@@ -3,8 +3,9 @@
  */
 
 resource "mysql_database" "schema" {
-  name              = "${var.schema_name}"
-  default_collation = "utf8_bin"
+  name                  = "${var.schema_name}"
+  default_character_set = "${lookup(var.optional_parameters, "default_character_set", "utf8")}"
```

Comment:
Those are the defaults for existing resource that didn't explicitly configure those 2 parameters right?

Reply:
yes